startapp.sh

```diff
@@ -1,15 +1,6 @@
 #!/bin/bash
 
-mkdir -p /tmp/borg
+groupadd -g 33 www-data
+adduser --no-create-home --quiet --uid 33 --gid 33 --disabled-login --force-badname --gecos www-data www-data
```

Comment:
maybe also copy the /etc/group and /etc/shadow files to a volume after the initial creation and then always copy it back and also before stopping save it in a volume... Same could be done for other important files